pokemongo_bot/__init__.py

```diff
@@ -763,11 +763,15 @@ def _setup_logging(self):
         if self.config.logging:
             logging_format = '%(message)s'
             logging_format_options = ''
-            if ('show_log_level' in self.config.logging and self.config.logging['show_log_level']) or 'show_log_level' not in self.config.logging:
+            
+            if 'show_log_level' not in self.config.logging or \
+                    ('show_log_level' in self.config.logging and self.config.logging['show_log_level']):
```

Comment:
Sorry, I know I'm being a nazi here. The checks in the parenthesis will evaluate first, as they have a higher evaluation precedence, even though python evaluates left to right. "in" has a lower precedence than attributes and indices. 

Please use the following (reduces the double "in" check too:
if ('show_log_level' not in self.config.logging) or self.config.logging['show_log_level']: